vars/iqeUtils.groovy

```diff
@@ -305,9 +312,13 @@ def runIQE(String plugin, Map appOptions) {
                 ),
                 returnStatus: true
             )
-            if (status > 0) {
```

Comment:
I'm not sure this logic change is correct. 
being the output of a bash script, I'd say exit code 0 is absolutely ok, and everything different than 0 (i.e. , > 0 ) is a failure. 

The change from status == 1 is "FAILURE", everything else is considered "ERROR" seems weird to me, probably this is a bug. ?

Reply:
@Victoremepunto I meant for this to be based on the exit codes of `pytest`, https://docs.pytest.org/en/documentation-restructure/how-to/usage.html#possible-exit-codes,
So `1` means that tests ran successfully and one or more failed. 

My intent was the the Jenkins pipeline should only have a `FAILURE` state when the tests actually fail, anything else should be considered an error state. However, I've clearly misunderstood the different states that Jenkins expects here.

Reply:
Thank you for the discussion, makes sense ! :+1: